app/queues/notifications/certificate-expiration-monitor-worker.server.ts

```diff
@@ -0,0 +1,53 @@
+import { Worker } from 'bullmq';
+import { redis } from '~/lib/redis.server';
+import logger from '~/lib/logger.server';
+import type { DomainStatusData } from './record-cert-expiration-data.interface';
+import { addNotification } from './notifications.server';
+import { prisma } from '~/db.server';
+
+// name for the queue
+export const certExpirationQueueName = 'notifications-certExpiration';
```

Comment:
See discussion in https://github.com/DevelopingSpace/starchart/pull/289#pullrequestreview-1320483880.  I think we can likely co-locate all notification logic into a single queue, since the process is going to be very similar in all cases (e.g., do a query, based on results create notification jobs).  The worker can use an if/switch to allow for slightly different logic in each case.